boards/esp32s3-seeedstudio/include/periph_conf.h

```diff
@@ -0,0 +1,167 @@
+ *
+ * All pins of ADC1 are declared as ADC channels. Keep in mind that GPIO10 is
+ * not broken out and used as VBAT signal. GPIO3 is a strapping pin if
+ * `EFUSE_STRAP_JTAG_SEL` is set, see \ref esp32_gpio_pins_esp32s3_strapping.
+ *
+ * @note As long as the GPIOs listed in ADC_GPIOS are not initialized as ADC
+ * channels with the `adc_init` function, they can be used for other
+ * purposes.
+ */
+#ifndef ADC_GPIOS
+#define ADC_GPIOS   { GPIO1, GPIO2, GPIO3, GPIO4, GPIO5, GPIO6, GPIO7, GPIO8, GPIO9 }
+#endif
+/** @} */
+
+/**
+ * @name   I2C configuration
+ *
+ * For generic boards, only one I2C interface I2C_DEV(0) is defined.
+ *
+ * The GPIOs listed in the configuration are only initialized as I2C signals
+ * when module `periph_i2c` is used. Otherwise they are not allocated and
+ * can be used for other purposes.
+ *
+ * @{
+ */
+#ifndef I2C0_SPEED
+#define I2C0_SPEED  I2C_SPEED_FAST  /**< I2C bus speed of I2C_DEV(0) */
+#endif
+#ifndef I2C0_SCL
+#define I2C0_SCL    GPIO6           /**< SCL signal of I2C_DEV(0) */
+#endif
+#ifndef I2C0_SDA
+#define I2C0_SDA    GPIO5           /**< SDA signal of I2C_DEV(0) */
+#endif
+/** @} */
+
+/**
+ * @name   PWM channel configuration
+ *
+ * Two PWM devices are configured. These devices can use all GPIOs that are not
+ * defined as I2C, SPI or UART for this board. Generally, all outputs pins
+ * could be used as PWM channels.
+ *
+ * @note As long as the according PWM device is not initialized with
+ * the `pwm_init`, the GPIOs declared for this device can be used
+ * for other purposes.
+ *
+ * @{
+ */
+
+/** TODO all gpios are pwm in seeed esp32s3
+ * @brief Declaration of the channels for device PWM_DEV(0),
+ *        at maximum PWM_CHANNEL_NUM_DEV_MAX.
+ */
+// #ifndef PWM0_GPIOS
+// #define PWM0_GPIOS  { GPIO12, GPIO13, GPIO14, GPIO15, GPIO16 }
+// #endif
+
+/**
+ * @brief Declaration of the channels for device PWM_DEV(1),
+ *        at maximum PWM_CHANNEL_NUM_DEV_MAX.
+ */
+// #ifndef PWM1_GPIOS
+// #define PWM1_GPIOS  { GPIO6, GPIO7, GPIO21, GPIO38 }
+// #endif
+
+/** @} */
+
+/**
+ * @name    SPI configuration
+ *
+ * @note The GPIOs listed in the configuration are first initialized as SPI
+ * signals when the corresponding SPI interface is used for the first time
+ * by either calling the `spi_init_cs` function or the `spi_acquire`
+ * function. That is, they are not allocated as SPI signals before and can
+ * be used for other purposes as long as the SPI interface is not used.
+ * @{ TODO SPI pins
+ */
+// #ifndef SPI0_CTRL
+// #define SPI0_CTRL   FSPI    /**< FSPI is used as SPI_DEV(0) */
+// #endif
+// #ifndef SPI0_SCK
+// #define SPI0_SCK    GPIO36  /**< FSPI SCK (pin FSPICLK) */
+// #endif
+// #ifndef SPI0_MISO
+// #define SPI0_MISO   GPIO37  /**< FSPI MISO (pin FSPIQ) */
+// #endif
+// #ifndef SPI0_MOSI
+// #define SPI0_MOSI   GPIO35  /**< FSPI MOSI (pin FSPID) */
+// #endif
+// #ifndef SPI0_CS0
+// #define SPI0_CS0    GPIO34  /**< FSPI CS0 (pin FSPICS0) */
+// #endif
+/** @} */
```

Comment:
Either define them or remove them :wink:

Reply:
I would prefer to define them at least for the SPI what schouldn't be a problem.

```
#define SPI0_SCK    GPIO7
...
#define SPI0_MISO   GPIO8
...
#define SPI0_MOSI   GPIO9
...
#define SPI0_CS0    <any available GPIO, e.g. GPIO6 (D3/A3)>
```

Reply:
Unless you define them, the doc.txt would need to be adapted to reflect the pins that are actually usable from RIOT with the default configuration.